.azure-pipelines.yml

```diff
@@ -142,7 +142,7 @@ jobs:
   variables:
     VCPKG_INSTALL_ROOT: $(Build.SourcesDirectory)\vcpkg
     VCPKG_ARCH: 'x64-windows'
-    VCPKG_PACKAGES: 'assimp boost-system boost-filesystem ccd eigen3 fcl'
+    VCPKG_PACKAGES: 'assimp boost-system boost-filesystem bullet3 ccd eigen3 fcl ode'
```

Comment:
Nit: Could we have a separate list for the optional dependencies something like `VCPKG_OPTIONAL_PACKAGES: 'bullet3 ode'` and add the variable at the end of line 154?

Reply:
ff53e8625a2bc48c2793ad8482b2a708f617f135